styles/search-input.scss

```diff
@@ -0,0 +1,26 @@
+.c-search-input {
+	border-radius: 3px;
+	position: relative;
+	z-index: 0;
+	overflow: hidden;
+
+	&::before {
+		content: "";
+		position: absolute;
+		inset: 0;
+		background: var(--color-white);
+		opacity: 0.1;
+	}
+}
+.c-search-input__input {
+	display: inline-block;
```

Comment:
Perhaps this class should have `display: block;`, but `.c-search-input` should have `display: inline-block;`